awx/main/tests/unit/models/test_job_template_unit.py

```diff
@@ -22,20 +23,6 @@ def test_inventory_credential_need_to_start(job_template_factory):
         persisted=False)
     obj = objects.job_template
     assert 'inventory' in obj.resources_needed_to_start
-    assert 'credential' in obj.resources_needed_to_start
-
-
-def test_inventory_credential_contradictions(job_template_factory):
-    objects = job_template_factory(
-        'job-template-paradox',
-        project='project1',
-        persisted=False)
-    obj = objects.job_template
-    obj.ask_inventory_on_launch = False
-    obj.ask_credential_on_launch = False
-    validation_errors, resources_needed_to_start = obj.resource_validation_data()
-    assert 'inventory' in validation_errors
-    assert 'credential' in validation_errors
```

Comment:
don't we still want coverage for this? What if the inventory is null and `ask_inventory_on_launch` is True? That's an invalid item

Reply:
Yep, that's a good point.   Looks like I got a little overzealous on this one - I'll restore and modify it.